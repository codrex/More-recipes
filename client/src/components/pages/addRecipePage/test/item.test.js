import React from 'react';
import Item from '../addItems/item';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureStore from '../../../../store/configStore';


describe('Item component ', () => {
  test('render as expected', () => {
    const component = renderer.create(
      <Provider store={configureStore()}>
        <Item name="directions" />
      </Provider>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
