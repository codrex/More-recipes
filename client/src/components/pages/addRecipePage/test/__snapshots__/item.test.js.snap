// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item component  render as expected 1`] = `
<div
  className="d-flex flex-column "
/>
`;

exports[`Item component  render as expected 2`] = `
<Item
  name="directions"
>
  <div
    className="d-flex flex-column "
  />
</Item>
`;
